sdk/synapse/synapse-artifacts/review/synapse-artifacts.api.md

```diff
@@ -1007,14 +1011,6 @@ export interface CloudError {
     target?: string;
 }
 
-// @public
-export interface CloudErrorAutoGenerated {
```

Comment:
This is a breaking change. r8? I am ok with it. (Since we can use CloudError instead of CloudErrorAutoGenerated). But, make sure introducing breaking change is ok

Reply:
Yeah, this is one of the changes we are taking in this month. All languages are taking it for example Python: https://github.com/Azure/azure-sdk-for-python/pull/17097/files#diff-72a3d1f774834b5782716e08d67a5a6fafb514244c16fdd514db3fdddc156639